### DISCUSSION TITLE
BERT4Rec performance

### DISCUSSION THREAD

**Discussion by Li-Muyang** [RUCAIBox/RecBole#1050] [Q&A] [answered]
Hi all, I'm trying to compare the performance of one of our model with BERT4Rec, however the performance of BERT4Rec on recbole is really nowhere near the reported performance in original paper. So I just want to ask if anyone have any experience with BERT4Rec on recbole, and what's your results like?

**Comment by Ethan-TZ** [comment, marked as answer]
@Li-Muyang Hello, thanks for your attention to RecBole!
The performance of the model is affected by many factors.Different machines,different dataset processing methods and different hyperparameters will obviously affect the performance of the model.You should first ensure that the details of dataset processing and hyperparameters are consistent with the original paper.In addition,some people have discussed the performance of BERT4Rec model before.You can refer to [issue 945](https://github.com/RUCAIBox/RecBole/issues/945).

**Reply by Li-Muyang** [reply]
Very helpful, many thanks!
